shared/constants/profile.js

```diff
@@ -56,8 +52,6 @@ export type OnClickFollowers = NoErrorTypedAction<'profile:onClickFollowers', {u
 export type OnClickFollowing = NoErrorTypedAction<'profile:onClickFollowing', {username: ?string, uid: string, openWebsite: ?boolean}>
 export type OnUserClick = NoErrorTypedAction<'profile:onUserClick', {username: string, uid: string}>
 export type OutputInstructionsActionLink = NoErrorTypedAction<'profile:outputInstructionsActionLink', void>
-export type RegisterBTC = NoErrorTypedAction<'profile:registerBTC', void>
```

Comment:
Yeah, weirdly. The action creators just navigate!?

https://github.com/keybase/client/pull/4857/files/d5e55f25e3e597657e781fa07ed6ed8f0cad408e#diff-5608bc4704557209a0af938fa87e6123L20

I'm a big confused how this worked in the first place. Am I missing something?